python/podio/frame.py

```diff
@@ -163,6 +196,46 @@ def _get_param_value(par_type, name):
 
     return _get_param_value(vec_types[0], name)
 
+  def put_parameter(self, key, value):
```

Comment:
Since the overload for floats will always be double, is it worth it to add a `float=True` that will select the float overload? Otherwise it's impossible to add a float parameter from python, in the rare case someone needs float and not double...

Reply:
Possible now via an additional `as_type` argument.